src/components/shared/DataTable/DataTable.tsx

```diff
@@ -96,11 +96,12 @@ const DataTable = <T extends IDefaultResource>({
       isSelected: selected.includes(row["ID"]),
       cells: currentColumns.map((column) => {
         const value = get(row, column.accessor, null)
+        const formattedValue = Array.isArray(value) ? value.join(", ") : value
```

Comment:
You should be able to do this with your cell renderer function without changing data table. However, I do see the need for having a default handler of rendering array values in a cell. Your formatted value will work for now, but instead of passing that into the `cell()` function as `value: formattedValue`, still pass in the original value so that `formattedValue` is only used as the default render.

Perhaps in a later update we can make the data table handle array values by creating a sub table by default unless otherwise handled by the cell renderer.

Reply:
Updated the code to only use `formattedValue` as the default render.